packages/utils/test/lighthouserc.test.js

```diff
@@ -13,23 +13,30 @@ const rc = require('../src/lighthouserc.js');
 const {safeDeleteFile} = require('../../cli/test/test-utils.js');
 
 describe('lighthouserc.js', () => {
+  function writeFile(path, content) {
```

Comment:
seems like this just passes the args directly to writeFileSync ;)

Reply:
🤦🏼‍♂️🤦🏼‍♂️🤦🏼‍♂️ what a silly mistake 😋.

Fixed.